support/artifacts/run.sh

```diff
@@ -40,9 +40,14 @@ fi
 git clone $repo_url output/augur-contracts
 current_dir=$PWD;
 cd output/augur-contracts
+
+git config user.email "team@augur.net"
+git config user.name "The Augur Developers"
```

Comment:
Consider setting the name to something like `Augur Artifact Scripts` or `Augur CI` so it is clear that this isn't coming from a person (and probably hasn't been looked at by a human).

Reply:
I like that!